src/igtcloud/client/tools/upload_project.py

```diff
@@ -32,6 +34,32 @@ def upload_project(local_folder: str, project_name: str, institute_name: str = N
     if submit:
         _password = getpass("For electronic record state it is required to reenter the password")
 
+    if category.lower() == "annotation":
+        # Annotation file upload
+        logger.info("Annotation File Uploading...")
+        if os.path.isdir(local_folder):
+            for root, dirs, local_files in os.walk(local_folder):
+                key = root
+                annotation_file = local_files
+        else:
+            logger.error("Not a valid Directory")
+        key = key + "/" + str(annotation_file[0])
+        logger.debug("Upload annotation path" + key)
+        try:
+            f = open(os.path.abspath(key), "r")
```

Comment:
The file is not being closed. Please open a file with a `with` statement: https://www.geeksforgeeks.org/with-statement-in-python/

Reply:
Added with open the file